assets/js/components/Household/create-household.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="household">
+        <input v-model="name"/>
```

Comment:
If you wrap this in a form tag like this:

```
<form @submit.prevent="createHousehold">
    <input v-model="name"/>
    <input type="submit" value="Create Household"/>
</form>
```

It has slightly better UX because hitting enter will trigger the submit action, in addition to clicking on the button

Reply:
Yep, I agree, but I am not really expecting anything but the `createHousehold()` method to ever be used, just needed the UI for debugging.

I will make this whole thing look pretty as soon as the backend can do everything we need it to do.